crates/subspace-farmer/src/single_plot_farm.rs

```diff
@@ -517,6 +518,23 @@ impl SinglePlotFarm {
                         })
                     }
                 }),
+                ObjectMappingsRequestHandler::create({
+                    let object_mappings = object_mappings.clone();
+
+                    move |&ObjectMappingsRequest { object_hash }| {
+                        let result = object_mappings.retrieve(&object_hash);
+                        let object_mapping = match result {
+                            Ok(res) => res,
+                            Err(err) => {
+                                error!(%err, ?object_hash, "Cannot retrieve an object mapping.");
```

Comment:
I think it should be `debug` here, otherwise it would be easy to flood logs.

Reply:
Do we expect errors here to be a common response?

Reply:
I think error is correct here, if mapping is unknown the response will be `None`, if error then something bad must have happened.

Reply:
I just think that we can flood logs with not actual errors (like internal protocol errors), but with user input errors

Reply:
This is a database read error. Unless database is corrupted or there is an implementation issue exists, we'll not see this in logs